hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/bucket/OMBucketSetPropertyRequest.java

```diff
@@ -168,12 +191,17 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
 
       omResponse.setSetBucketPropertyResponse(
           SetBucketPropertyResponse.newBuilder().build());
-      omClientResponse =
-          new OMBucketSetPropertyResponse(omBucketInfo, omResponse.build());
+      omClientResponse = new OMBucketSetPropertyResponse(
+          omResponse.build(), omBucketInfo);
+      LOG.debug("Setting bucket property for bucket:{} in volume:{}",
+          bucketName, volumeName);
     } catch (IOException ex) {
+      LOG.error("Setting bucket property failed for bucket:{} in volume:{}",
```

Comment:
Same here, can we move logging outside the lock.

Reply:
done.